frontend/src/metabase-lib/fields.ts

```diff
@@ -19,3 +19,14 @@ export function fieldableColumns(
 ): ColumnMetadata[] {
   return ML.fieldable_columns(query, stageIndex);
 }
+
+/**
+ * This should only be used to get field IDs when it is necessary, like interacting with backend API parameters.
+ * For most purposes, you should be use columnMetadata objects and not access field ids directly
+ *
+ * @param {ColumnMetadata} column
+ * @returns { number } field id
+ */
+export function _fieldId(column: ColumnMetadata): number {
+  return ML.field_id(column);
+}
```

Comment:
`field_id` returns `null` given a calculated column, could you please include the `null` return type?

```suggestion
 * @returns {number|null} field id
 */
export function _fieldId(column: ColumnMetadata): number | null {
  return ML.field_id(column);
}
```